src/client-e2e-test/runCommand.ts

```diff
@@ -0,0 +1,7 @@
+import * as vscode from "vscode";
+import { cursorlessCommandId } from "../common/commandIds";
+import { Command } from "../core/commandRunner/command.types";
```

Comment:
This should eventually move into `common`, but maybe doesn't have to happen in this PR.  At the very least I'd do an `import type`